src/gql/generated/types.ts

```diff
@@ -1212,12 +1211,76 @@ export type GetTaskEventDataQueryVariables = Exact<{
 export type GetTaskEventDataQuery = {
   task?: Maybe<{
     id: string;
-    execution: number;
+    execution?: Maybe<number>;
```

Comment:
i think the `schema.graphql` you generated this with is out of date this field is always there. Pull the latest evergreen master and run `yarn codegen` again